documentation/docs/specs/did/iota_did_method_spec.md

```diff
@@ -58,246 +85,137 @@ base-char = 1-9 A-H J-N P-Z a-k m-z
 
 ### IOTA-Network
 
-The iota-network is an identifier of the network where the DID is stored. This network must be an IOTA Tangle, but can either be a public or private network, permissionless or permissioned.
+**TODO should be replaced by shimmer? **
+
+The iota-network is an identifier of the network where the DID is stored. This network must be an IOTA Ledger, but can either be a public or private network, permissionless or permissioned.
 
 The following values are reserved and cannot reference other networks:
+
 1. `main` references the main network which refers to the Tangle known to host the IOTA cryptocurrency.
 2. `dev` references the development network known as "devnet" maintained by the IOTA Foundation.
 
 When no IOTA network is specified, it is assumed that the DID is located on the `main` network. This means that the following DIDs will resolve to the same DID Document:
+
 ```
-did:iota:main:H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV
-did:iota:H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV
+did:iota:main:0xe4edef97da1257e83cbeb49159cfdd2da6ac971ac447f233f8439cf29376ebfe
+did:iota:0xe4edef97da1257e83cbeb49159cfdd2da6ac971ac447f233f8439cf29376ebfe
 ```
 
 ### IOTA-Tag
```

Comment:
We need to come up with a different term. The did spec calls it "method specific-identifier". Other suggestions:
`identifier`
`iota-id`

Reply:
We use `tag` in the framework though. The `method-specific-identifier` includes the `network` (e.g. `main:0x123`, `dev:0x123`, `rms:0x123`, etc.), so tag distinguishes "just the end bit"?

Reply:
Good point, since there is no obvious replacement, let's leave it as is.